src/dbus_api/types.rs

```diff
@@ -44,22 +44,13 @@ pub type InterfacesRemoved = Vec<String>;
 pub enum DbusErrorEnum {
     OK = 0,
     ERROR = 1,
-
-    ALREADY_EXISTS = 2,
-    BUSY = 3,
-    INTERNAL_ERROR = 4,
-    NOTFOUND = 5,
 }
 
 impl DbusErrorEnum {
```

Comment:
I think the enum makes sense just because it allows us to define constants in a logically related way. I can absolutely remove the `msg_code_ok()` and `msg_string_ok()` methods though. `msg_string_ok()` is a little bit more complicated because we'd probably want to define it as a constant instead and that requires calling `.to_string()` on a static `str` which would require `lazy_static`. What are your thoughts on that?

Reply:
What about implementing ```Display``` on the ```DbusErrorEnum``` and getting ```to_string()``` for free? ```DbusErrorEnum::OK.to_string()``` isn't so much worse, actually better, than calling ```msg_string_ok()``` as far as I'm concerned. Also more consistent with how ```DbusErrorEnum::ERROR as u16``` is used everywhere.

This suggestion isn't the only possiblity. Just any way you can see to get rid of some more of the cruft that is associated with ```DbusErrorEnum``` is fine w/ me.

Reply:
The reason I haven't implemented the display trait is that we'd never really use the `ERROR` variant in the enum. If you'd prefer explicitly calling `.to_string()` each time, I think a constant for the `Ok` message would work better.